preset/src/shortcuts/badge.ts

```diff
@@ -0,0 +1,15 @@
+export const badgeShortcuts: Record<string, string> = {
+	// badge
+	'k-badge--base': 'relative v-mid inline-block w-fit',
+	'k-badge--content':
+		'inline-flex justify-center items-center h-18px b-rd-10px text-12px p-y-0 p-x-6px whitespace-nowrap b-1px b-solid b-ikun-white color-ikun-white',
```

Comment:
You can add an `inline-flex justify-center items-center` combination in `shortcuts/common.ts`, just like `infsc`